src/blocks/SliderOld/Arrow/Arrow.tsx

```diff
@@ -7,24 +7,22 @@ import {i18n} from '../i18n';
 
 import './Arrow.scss';
 
-const b = block('slider-new-block-arrow');
+const b = block('slider-old-block-arrow');
 
 export type ArrowType = 'left' | 'right';
 
 export interface ArrowProps {
     type: ArrowType;
-    onClick?: () => void;
+    handleClick?: (direction: ArrowType) => void;
     size?: number;
-    extraProps?: React.ButtonHTMLAttributes<HTMLButtonElement>;
 }
 
-const Arrow = ({type, onClick, className, size = 16, extraProps}: ArrowProps & ClassNameProps) => (
+const Arrow = ({type, handleClick, className, size = 16}: ArrowProps & ClassNameProps) => (
     <div className={b({type}, className)}>
         <button
             className={b('button')}
-            onClick={onClick}
+            onClick={() => handleClick && handleClick(type)}
```

Comment:
maybe `handleClick ? () => handleClick() : null`?

Reply:
changed to `handleClick ? () => handleClick(type) : undefined`